### PR TITLE
latitude_from_zenith_distances_and_declinations

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: test/astronomy/latitude_from_zenith_distances_and_declinations_test.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## If the zenith distance and declination of the northern luminary are equal to 30 and 25 degrees, respectively,
+## and the zenith distance and declination of the southern luminary are equal to 150 and -15 degrees, respectively,
+## then the latitude is 65 degrees.
+
+Args = namedtuple("Args", ["zenith_distance_north", "zenith_distance_south", "declination_north", "declination_south",])
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    zenith_distance_north = 30
```
Comment (blackyblack) [inline test/astronomy/latitude_from_zenith_distances_and_declinations_test.py:16]: Use proper quantities.

File: symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    validate_output,
+    convert_to,
+    angle_type,
+)
+
+# Description
+## Knowing the zenith distances and declinations of the northern and southern stars, respectively, it is possible
+## to determine the latitude of the observation site.
+
+## Law is: phi = (zs - zn + ds + dn) / 2, where
```
Comment (blackyblack) [inline symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py:17]: Do you have a link to this law? I'm not sure how the stars should be chosen.

File: test/astronomy/latitude_from_zenith_distances_and_declinations_test.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors)
+from symplyphysics.laws.astronomy import latitude_from_zenith_distances_and_declinations as latitude_law
+
+# Description
+## If the zenith distance and declination of the northern luminary are equal to 30 and 25 degrees, respectively,
```
Comment (blackyblack) [inline test/astronomy/latitude_from_zenith_distances_and_declinations_test.py:7]: What is northern luminary? I cannot find this term on Google.

File: symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py (hunk 2 above)
Comment (Zulusssss) [reply symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py:17]: > Do you have a link to this law? I'm not sure how the stars should be chosen.

https://ru.wikipedia.org/wiki/Способ_Талькотта#cite_note-:0-3

File: test/astronomy/latitude_from_zenith_distances_and_declinations_test.py (hunk 3 above)
Comment (blackyblack) [reply test/astronomy/latitude_from_zenith_distances_and_declinations_test.py:7]: Use `star north of the zenith` and `star south of the zenith` instead.

File: symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py (hunk 2 above)
Comment (blackyblack) [reply symplyphysics/laws/astronomy/latitude_from_zenith_distances_and_declinations.py:17]: Mention that northern star is any star north of the zenith with known declination. Same for southern star.
Also add to the condtions that both stars are at upper transit (culmination).

**Review by blackyblack — APPROVED** [review]

**Pull request merged**